File: UtlaggshanteringBackendAPI/Models/UserExpense.cs
namespace Utläggningshantering.Models
{
    public class UserExpense
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string? PlaceOfPurchase { get; set; }
        public decimal AmountIncludingVAT { get; set; }
        public decimal VAT { get; set; }
        public string? Reason { get; set; }
        public string? Members { get; set; }
        public string? Comment { get; set; }
    }
}

File: UtlaggshanteringFrontend/src/app/models/Userexpense.ts
// Interface for UserExpense, added editable variable for extended functionality
export interface UserExpense{
    id?: number,
    date?: Date,
    placeOfPurchase?: string,
    amountIncludingVAT?: number,
    vat?: number,
    reason?: string,
    members?: string,
    comment?: string,
    editable?: boolean 
}